package.json

```diff
@@ -96,9 +96,9 @@
     "webpack-node-externals": "^2.5.2"
   },
   "dependencies": {
-    "@esri/proj-codes": "^2.7.0",
     "@godaddy/terminus": "^4.6.0",
-    "@openforis/arena-server": "^0.0.16",
+    "@openforis/arena-core": "^0.0.51",
```

Comment:
is this needed?

Reply:
yes, otherwise you cannot import modules from arena-core